Source/JavaScriptCore/assembler/AssemblerCommon.h

```diff
@@ -323,6 +323,15 @@ static void* performJITMemcpy(void *dst, const void *src, size_t n);
 template<MachineCodeCopyMode copy>
 ALWAYS_INLINE void* machineCodeCopy(void *dst, const void *src, size_t n)
 {
+    bool isAligned = !(reinterpret_cast<uintptr_t>(dst) & 3);
+    if (n == 2 * sizeof(int16_t) && isAligned) {
+        *static_cast<uint32_t*>(dst) = *static_cast<const uint32_t*>(src);
+        return dst;
+    }
+    if (n == 1 * sizeof(int16_t)) {
+        *static_cast<uint16_t*>(dst) = *static_cast<const uint16_t*>(src);
+        return dst;
+    }
```

Comment:
This part is AssemblerCommon, so if we add this here, we need to handle `performJITMemcpy` case too.
Can you wrap it with `CPU(ARM_THUMB2)` since this is not correct for `performJITMemcpy` case?

Reply:
:fearful:  Oops!